ci-operator/step-registry/sippy-staging/deploy/sippy-staging-deploy-commands.sh

```
@@ -1,7 +1,36 @@
+        # shellcheck disable=SC2086
+        kill ${children} 2>/dev/null || true
+        for i in $(seq 1 10); do
+            children="$(jobs -p 2>/dev/null || true)"
+            [[ -z "${children}" ]] && break
+            sleep 0.5
+        done
+        children="$(jobs -p 2>/dev/null || true)"
+        if [[ -n "${children}" ]]; then
+            # shellcheck disable=SC2086
+            kill -9 ${children} 2>/dev/null || true
+            for i in $(seq 1 5); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)